tests/by-util/test_seq.rs

```diff
@@ -816,3 +816,73 @@ fn test_parse_scientific_zero() {
         .succeeds()
         .stdout_only("0\n1\n");
 }
+
+#[test]
+fn test_parse_valid_hexadecimal_float() {
+    new_ucmd!()
+        .args(&["0x1p-1", "2"])
+        .succeeds()
+        .stdout_only("0.5\n1.5\n");
+
+    new_ucmd!()
+        .args(&["1", "0x1p-1", "2"])
+        .succeeds()
+        .stdout_only("1.0\n1.5\n2.0\n");
+
+    new_ucmd!()
+        .args(&["0x3.4p-1", "0x4p-1", "4"])
+        .succeeds()
+        .stdout_only("1.625\n3.625\n");
+
+    new_ucmd!()
+        .args(&["0x3.4p-1", "0x4p-1", "4"])
+        .succeeds()
+        .stdout_only("1.625\n3.625\n");
+
+    new_ucmd!()
+        .args(&[" 0x.8p16", "32768"])
+        .succeeds()
+        .stdout_only("32768\n");
+}
+
+#[ignore]
+#[test]
+fn test_parse_valid_hexadecimal_float_format_issues() {
+    // Parsing is OK, but value representation conflicts with the GNU seq.
+
+    // Test output: 4095.953125
+    // In fact, the 4095.953125 is correct value.
+    // (65535 + 4/16)*2^(-4) = 4095.953125
+    // So, it looks like a formatting or output rounding differences
+    new_ucmd!()
```

Comment:
@sylvestre Hello,

Could you please find some time to discuss the test results?

Here are the tests that showed differences in the output between the Rust and GNU versions of the utility.

The GNU seq uses ```printf``` with the ```%Lg``` format, which doesn't seem to have a direct equivalent in Rust. Furthermore, ```%Lg``` can switch between ```%f``` and ```%e``` formats depending on the precision, which also behaves differently in Rust.

I plan to experiment with formats and GNU glibc to replicate the printf behavior in Rust, but it might take some time. Please let me know if implementing a Rust equivalent is the preferred way to handle these output mismatches. 

Additionally, I was wondering if it might make sense to prepare and merge what we already have. This way, we can share the code and gather more real-world feedback. Since these changes address the initial issue, it’s still a step forward. Of course, I’m not insisting—just considering other options. I’m happy to proceed with whichever approach works best for the project.

Thx

Reply:
>  Please let me know if implementing a Rust equivalent is the preferred way to handle these output mismatches.

Looks good to me :)

Reply:
I will move in that direction. Thank you